addons/account/wizard/account_invoice_send.py

```diff
@@ -116,7 +116,7 @@ def _compute_invoice_without_email(self):
     def _send_email(self):
         if self.is_email:
             # with_context : we don't want to reimport the file we just exported.
-            self.composer_id.with_context(no_new_invoice=True, mail_notify_author=self.env.user.partner_id in self.composer_id.partner_ids)._action_send_mail()
+            self.composer_id.with_context(no_new_invoice=True, mail_notify_author=self.env.user.partner_id in self.composer_id.partner_ids, document_based=True)._action_send_mail()
```

Comment:
Linting: could be multi-lined to ease reading (beginning to explode outside of my screen ^^)